src/androidTest/java/de/dennisguse/opentracks/services/announcement/VoiceAnnouncementUtilsTest.java

```diff
@@ -91,10 +120,10 @@ public void getAnnouncement_metric_pace() {
         stats.setTotalAltitudeGain(6000f);
 
         // when
-        String announcement = VoiceAnnouncementUtils.getAnnouncement(context, stats, true, false, null);
+        String announcement = VoiceAnnouncementUtils.getAnnouncement(context, stats, true, false, null, null).toString();
 
         // then
-        assertEquals("total distance 20.00 kilometers in 1 hour 5 minutes 10 seconds at 3 minutes 15 seconds per kilometer", announcement);
+        assertEquals("Total distance 20.00 kilometers, 1 hour 5 minutes 10 seconds, Pace 3 minutes 15 seconds per kilometer,", announcement);
```

Comment:
Comma at the end is because all append statements insert a comma at the end. It makes no difference to TTS output and this text is never displayed anywhere.